docs/faq.md

```diff
@@ -24,8 +24,66 @@ https://kind.sigs.k8s.io/docs/user/quick-start/
 [Docker Desktop]:
 https://docs.docker.com/desktop/
 
-**Q: Can I have Docker Desktop installed alongside Rancher Desktop?**
+<!-- #1221 -->
+**Q: Can I run Rancher Desktop and Docker Desktop simultaneously?**
 
-**A:** Yes, there are no conflicts having both installed.
+**A:** No, they cannot be run at the same time as both Rancher Desktop and Docker Desktop use the same Docker socket (`/var/run/docker.sock`). Both can be installed and run, provided you stop one before starting the other.
```

Comment:
The question changed from "can I install" to "can I run" and the answer went from Yes to No. We want people to try RD out and not think they may have to uninstall DD to do that.

What about flipping it around so that it's "can I install" with an answer of "yes" but a note that you can't run them both at the same time.

Reply:
Updated.